### PR TITLE
Stream pallet changes: companion

### PR DESCRIPTION
**Pull request by amarts**
companion for https://github.com/dhiway/cord/pull/241

### CODE REVIEW

File: packages/messaging/src/Message.ts [hunk 1]
```diff
@@ -150,10 +150,12 @@ export function ensureOwnerIsSender({ body, sender }: IMessage): void {
       break
     case 'submit-document-stream':
       {
+	/*
```
Comment (amarts) [inline packages/messaging/src/Message.ts:153]: this is done as document would be taken out and we would bring VC as a native type inside of SDK.

File: packages/modules/src/stream/Stream.chain.ts [hunk 2]
```diff
@@ -51,11 +52,24 @@ export function fromChain(
...
-    issuer: Did.fromChain(chainStream.creator),
     schema: DecoderUtils.hexToString(chainStream.schema.toString()),
     registry: DecoderUtils.hexToString(chainStream.registry.toString()) || null,
-    revoked: chainStream.revoked.valueOf(),
   }
   log.info(`Decoded stream: ${JSON.stringify(stream)}`)
   return stream
 }
+
+export function fromChainAttest(
```
Comment (smohan-dw) [inline packages/modules/src/stream/Stream.chain.ts:62]: Here the query should have an identifier and digest, right?

Comment (smohan-dw) [reply packages/modules/src/stream/Stream.chain.ts:62]: my wrong, the flow is correct. you are only decoding the result.

**Review by smohan-dw — APPROVED** [review]
LGTM

**Pull request merged**